test/TesterInternal/OrleansRuntime/ManagementGrainTests.cs

```diff
@@ -0,0 +1,42 @@
+using System.Threading.Tasks;
+using Orleans.Runtime;
+using Orleans.TestingHost;
+using TestExtensions;
+using Xunit;
+
+namespace UnitTests.OrleansRuntime
+{
+    public class ManagementGrainTests : OrleansTestingBase, IClassFixture<ManagementGrainTests.Fixture>
+    {
+        private readonly Fixture fixture;
+
+        public class Fixture : BaseTestClusterFixture
+        {
+            protected override void ConfigureTestCluster(TestClusterBuilder builder)
+            {
+                builder.Options.InitialSilosCount = 2;
+            }
+        }
+
+        public ManagementGrainTests(Fixture fixture)
+        {
+            this.fixture = fixture;
+        }
+
+        [Fact]
```

Comment:
Set `TestCategory("BVT")` or it will not be run in CI

Reply:
Sorry, forgot that. Its being a while. 😅